Add class and id to make lesson text readable.

eschlange opened the pull request:
[#95493138]

michaelwehrley commented on app/assets/stylesheets/sunnyside.css.scss:
```diff
@@ -60,7 +60,7 @@ button.home-toggle {
 
-.tool-content h3.lesson-list-text, .lesson-list-text, .lesson-list-text span {
+.tool-content h3.lesson-list-text, .lesson-list-text, .lesson-list-text span, #tool-layout h3.panel-title, .tool-content h3.panel-title {
   color: #aaaaaa !important;
```
is it reasonable to nest the css so we don't need the `!important`?  I hate `!important`, but may be a critique on style over code.  If this feeling is overkill, then :+1: :ship: it!

eschlange replied:
@michaelwehrley I feel the same way. Due to how we are utilizing code from the engine and overwriting it in the host application, I was forced to use this. I can't get around this without updating the engine code, which I thought would be bad form since it would affect other host application views.

eschlange replied:
@michaelwehrley Are you okay with me merging this code, given this circumstance?

michaelwehrley replied:
Would it make sense to change the css order: https://github.com/cbitstech/sunnyside/blob/master/app/assets/stylesheets/application.css.scss#L9-11 Would that help?

eschlange commented:
@michaelwehrley Thanks for pointing that out! I was wondering why I was having so many style conflicts. I updated the import order as you suggested and removed the "important!" syntax.

michaelwehrley commented:
Hells yeah! :+1: :shipit: :pizza:
